src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { BrowserRouter, Switch, Route } from "react-router-dom";
+import {Helmet} from 'react-helmet';
+
+// import application components
+import ContactForm from "./ContactForm/ContactForm";
```

Comment:
Wondering why it is called contact form? Might be better to name it in a way that matches the expected output, i.e. Basics Form?

Reply:
good point